### ISSUE TITLE
it's black screen wehn run release binary

### ISSUE THREAD

**Issue by zinwalin** [Nadrin/quartz#5]
anyone had the same issue? 

![image](https://user-images.githubusercontent.com/5416196/69400672-6a99c680-0d57-11ea-89e2-0e5e58e5e5b8.png)

**Comment by PerspectivesLab** [comment]
yes i have same, nvidia quadro p4000, black screen, i dont know how to resolve this... any hint please ?

**Comment by HorstBaerbel** [comment]
As far as I understand it, you need a Vulkan device + driver that supports the "VK_NV_raytracing" extension, which means a card that has raytracing cores, like the Nvidia RTX series.

**Comment by SirKnightTG** [comment]
I have this issue with an RTX 2080 Ti and newest driver. This used to work, but I'm thinking a driver update at some point caused the breakage. It could be the app is relying on a bug in the driver that was fixed.

I had some issues getting the code to build, but it's been a while so I don't remember what was going on. I should try again.

**Comment by SirKnightTG** [comment]
> As far as I understand it, you need a Vulkan device + driver that supports the "VK_NV_raytracing" extension, which means a card that has raytracing cores, like the Nvidia RTX series.

You don't need RTCores for VK_NV_raytracing (or DXR) support to exist. Pascal going back to 1060 will work. It'll be slow as heck, but it'll work.

**Comment by HorstBaerbel** [comment]
> You don't need RTCores for VK_NV_raytracing (or DXR) support to exist. Pascal going back to 1060 will work. It'll be slow as heck, but it'll work.

Thanks for the clarification. Didn't know that. Now I feel tempted to try it on my GTX 1650 Ti... ;)

**Comment by jcxz** [comment]
What is the status of this issue ? I am also seeing only black screen.
